### PR TITLE
Add docs about default callback implementation

### PR DESCRIPTION
**Pull request by Rados13**
Solves: https://membraneframework.atlassian.net/browse/MSB-63?atlOrigin=eyJpIjoiZWUwYzgzNmJhNDI1NDZiYmI0YjNmZTI1ZjYyYzQzYmEiLCJwIjoiaiJ9

### CODE REVIEW

File: lib/membrane/element/base.ex [hunk 1]
```diff
@@ -45,6 +45,7 @@ defmodule Membrane.Element.Base do
   that happen in this callback crash the parent as well, regardless of crash groups.
   For these reasons, it's important to do any long-lasting or complex work in `c:handle_setup/2`,
   while `handle_init` should be used for things like parsing options or initializing state.
+  By default it create map from opts struct.
```
Comment (FelonEkonom) [inline lib/membrane/element/base.ex:48]:
```suggestion
  By default it creates map from opts struct.
```
Here and in Bin

File: lib/membrane/pipeline.ex [hunk 2]
```diff
@@ -151,6 +155,7 @@ defmodule Membrane.Pipeline do
 
   @doc """
   Callback invoked when a notification comes in from an element.
+  By default it ignores received notification from child.
```
Comment (FelonEkonom) [inline lib/membrane/pipeline.ex:158]:
```suggestion
 By default, it ignores the notification received from the child.
```

File: lib/membrane/pipeline.ex [hunk 3]
```diff
@@ -164,6 +169,7 @@ defmodule Membrane.Pipeline do
   as an internal membrane message.
 
   Useful for receiving data sent from NIFs or other stuff.
+  By default it ignores received message.
```
Comment (FelonEkonom) [inline lib/membrane/pipeline.ex:172]:
```suggestion
 By default, it ignores the received message.
```

**Review by FelonEkonom — CHANGES_REQUESTED** [review]

**Review by FelonEkonom — APPROVED** [review]

File: lib/membrane/bin.ex [hunk 4]
```diff
@@ -136,6 +145,7 @@ defmodule Membrane.Bin do
 
   @doc """
   Callback invoked when a child element starts processing stream via given pad.
+  By default it ignores `:start_of_stream`.
```
Comment (mat-hek) [inline lib/membrane/bin.ex:148]:
```suggestion
  
  By default it does nothing.
```

File: lib/membrane/bin.ex [hunk 5]
```diff
@@ -146,6 +156,7 @@ defmodule Membrane.Bin do
 
   @doc """
   Callback invoked when a child element finishes processing stream via given pad.
+  By default it ignores `:end_of_stream`.
```
Comment (mat-hek) [inline lib/membrane/bin.ex:159]:
```suggestion

  By default it does nothing.
```

File: lib/membrane/bin.ex [hunk 6]
```diff
@@ -156,6 +167,7 @@ defmodule Membrane.Bin do
 
   @doc """
   Callback invoked when children of `Membrane.ChildrenSpec` are started.
+  By default, it does nothing.
```
Comment (mat-hek) [inline lib/membrane/bin.ex:170]:
```suggestion

  By default, it does nothing.
```

File: lib/membrane/element/base.ex [hunk 7]
```diff
@@ -75,6 +78,7 @@ defmodule Membrane.Element.Base do
   as an internal membrane message.
 
   Useful for receiving ticks from timer, data sent from NIFs or other stuff.
+  By default it ignores message.
```
Comment (mat-hek) [inline lib/membrane/element/base.ex:81]:
```suggestion
  By default it ignores the received message.
```

File: lib/membrane/element/base.ex [hunk 8]
```diff
@@ -131,6 +138,7 @@ defmodule Membrane.Element.Base do
 
   @doc """
   Callback invoked when a message from the parent is received.
+  By default it ignores received message from parent.
```
Comment (mat-hek) [inline lib/membrane/element/base.ex:141]:
```suggestion
  By default it ignores the received message.
```

File: lib/membrane/pipeline.ex [hunk 9]
```diff
@@ -151,6 +155,7 @@ defmodule Membrane.Pipeline do
 
   @doc """
   Callback invoked when a notification comes in from an element.
+  By default, it ignores the notification received from the child.
```
Comment (mat-hek) [inline lib/membrane/pipeline.ex:158]:
```suggestion
  Callback invoked when a notification comes in from a child.

  By default, it ignores the notification.
```

File: lib/membrane/pipeline.ex [hunk 10]
```diff
@@ -174,6 +180,7 @@ defmodule Membrane.Pipeline do
 
   @doc """
   Callback invoked when a child element starts processing stream via given pad.
+  By default it ignores `:start_of_stream`.
```
Comment (mat-hek) [inline lib/membrane/pipeline.ex:183]:
```suggestion
  
  By default it does nothing.
```

File: lib/membrane/pipeline.ex [hunk 11]
```diff
@@ -184,6 +191,7 @@ defmodule Membrane.Pipeline do
 
   @doc """
   Callback invoked when a child element finishes processing stream via given pad.
+  By default it ignores `:end_of_stream`.
```
Comment (mat-hek) [inline lib/membrane/pipeline.ex:194]:
```suggestion

  By default it does nothing.
```

File: lib/membrane/pipeline.ex [hunk 12]
```diff
@@ -194,6 +202,7 @@ defmodule Membrane.Pipeline do
 
   @doc """
   Callback invoked when children of `Membrane.ChildrenSpec` are started.
+  By default, it does nothing.
```
Comment (mat-hek) [inline lib/membrane/pipeline.ex:205]:
```suggestion

  By default, it does nothing.
```

File: lib/membrane/pipeline.ex [hunk 13]
```diff
@@ -103,6 +103,7 @@ defmodule Membrane.Pipeline do
   finishes. For that reason, it's important to do any long-lasting or complex work in `c:handle_setup/2`,
   while `handle_init` should be used for things like parsing options, initializing state or spawning
   children.
+  By default, it creates a map from opts struct.
```
Comment (mat-hek) [inline lib/membrane/pipeline.ex:106]:
```suggestion
  By default, it converts the `opts` to a map if they're a struct and sets them as the pipeline state.
```

File: lib/membrane/bin.ex [hunk 14]
```diff
@@ -42,6 +42,7 @@ defmodule Membrane.Bin do
   For these reasons, it's important to do any long-lasting or complex work in `c:handle_setup/2`,
   while `handle_init` should be used for things like parsing options, initializing state or
   spawning children.
+  By default, it creates a map from opts struct.
```
Comment (mat-hek) [inline lib/membrane/bin.ex:45]: By default, it converts the `opts` struct to a map and sets them as the bin's state.

File: lib/membrane/element/base.ex [hunk 15]
```diff
@@ -45,6 +45,7 @@ defmodule Membrane.Element.Base do
   that happen in this callback crash the parent as well, regardless of crash groups.
   For these reasons, it's important to do any long-lasting or complex work in `c:handle_setup/2`,
   while `handle_init` should be used for things like parsing options or initializing state.
+  By default, it creates a map from opts struct.
```
Comment (mat-hek) [inline lib/membrane/element/base.ex:48]:
```suggestion
  By default, it converts the `opts` struct to a map and sets them as the element's state.
```

File: lib/membrane/bin.ex [hunk 16]
```diff
@@ -156,6 +168,10 @@ defmodule Membrane.Bin do
 
   @doc """
   Callback invoked when children of `Membrane.ChildrenSpec` are started.
+
+
```
Comment (mat-hek) [inline lib/membrane/bin.ex:172]:
```suggestion
```

**Review by mat-hek — APPROVED** [review]

**Pull request merged**